submissions/oluwakemiRichard/HTML/index.html

```diff
@@ -14,7 +14,7 @@
                 <div class="searchField">
                 <input type="text" placeholder="search.." name="search">
                 <button type="submit">Submit</button>
-                </div>
+                </div>git push origin contact_form
```

Comment:
Is this intentional `git push origin contact_form`?

Reply:
No. It is not. I have deleted it.